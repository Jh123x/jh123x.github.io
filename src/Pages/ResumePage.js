import React from 'react';
import Skills from '../Components/Skills';
import { MainLayout} from '../styles/Layouts';
import Resume from '../Components/Resume';

/**
 * ResumePage Component is the resume page of the website.
 * @returns {React.JSX.Element}
 */
const ResumePage = () => {
    return (
        <MainLayout>
            <Skills />
            <Resume />
        </MainLayout>
    )
}

export default ResumePage
