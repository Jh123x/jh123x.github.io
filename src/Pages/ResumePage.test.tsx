import * as React from "react";
import ResumePage from "./ResumePage";
import { describe, expect, test } from "@jest/globals";

describe("ResumePage Page", () => {
  test("should render correctly", () => {
    const page = <ResumePage />;
    expect(page).toMatchSnapshot();
  });
});
