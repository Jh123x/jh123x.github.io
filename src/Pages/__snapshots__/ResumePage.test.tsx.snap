// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResumePage Page should render correctly 1`] = `<ResumePage />`;
